contracts/VaultV2.sol

```diff
@@ -47,7 +47,7 @@ contract VaultV2 is ERC20("Tracer Vault Token", "TVT", 18), IERC4626, Ownable {
             sumPercent += percent;
             percentAllocations.push(percent);
         }
-        require(sumPercent <= BASE_UNIT, "PERC_SUM_MAX");
+        require(sumPercent <= baseUnit, "PERC_SUM_MAX");
```

Comment:
Not part of this PR but I think `baseUnit` should be prepended with an underscore to remain consistent with the other function variables.